mtp_api/apps/transaction/models.py

```diff
@@ -0,0 +1,27 @@
+from django.db import models
+
+from model_utils.models import TimeStampedModel
+
+from prison.models import Prison
+
+
+class Transaction(TimeStampedModel):
+    upload_counter = models.PositiveIntegerField()
+
+    prison = models.ForeignKey(Prison, blank=True, null=True)
+
+    prisoner_number = models.CharField(blank=True, max_length=250)
+    prisoner_name = models.CharField(blank=True, max_length=250)
+    prisoner_dob = models.DateField(blank=True, null=True)
+
+    # credit_or_debit = models.CharField(max_length=50)
+    amount = models.PositiveIntegerField()
+    sender_bank_reference = models.CharField(
+        blank=True, max_length=250
+    )
+    sender_customer_reference = models.CharField(
+        blank=True, max_length=250
+    )
+    reference = models.TextField(blank=True)
+    received_at = models.DateTimeField(auto_now=False)
+    # raw = models.TextField()
```

Comment:
Was keeping them there as informal documentation as we will need them in the future but I didn't want to add them right now as they would be annoying to manage.
I will delete them anyway.